dev/flipbook.tsx

```diff
@@ -83,30 +89,63 @@ type QuestionRendererProps = {
     apiOptions?: APIOptions;
 };
 
-function QuestionRenderer({question, apiOptions = {}}: QuestionRendererProps) {
+function SideBySideQuestionRenderer({
+    question,
+    apiOptions = {},
+}: QuestionRendererProps) {
     return (
-        <div
+        <View
+            className="framework-perseus"
             style={{
+                flexDirection: "row",
                 padding: Spacing.xLarge_32,
-                display: "flex",
                 gap: Spacing.small_12,
             }}
-            className="framework-perseus"
         >
-            <Renderer
-                content={question.content}
-                images={question.images}
-                widgets={question.widgets}
-                problemNum={0}
+            <GradableRenderer
+                question={question}
                 apiOptions={{...apiOptions, flags: {mafs: false}}}
             />
+            <GradableRenderer
+                question={question}
+                apiOptions={{...apiOptions, flags: {mafs: {segment: true}}}}
+            />
+        </View>
+    );
+}
+
+function GradableRenderer(props: QuestionRendererProps) {
+    const {question, apiOptions} = props;
+    const rendererRef = useRef<Renderer>(null);
+
+    function describeScore(score: PerseusScore): string {
+        switch (score.type) {
+            case "invalid":
+                return "You didn't answer the question.";
+            case "points":
+                return score.earned > 0 ? "Correct!" : "Incorrect.";
```

Comment:
This is not quite right. For a multi-widget question, the `total` field on the score may be more than 1 and so to check for correctness, we should be using `score.earned >= score.total` (I'm not sure how it could be MORE, but that's how https://github.com/Khan/perseus/blob/881da724d36561b2a200af0258ab81523d4d236a/packages/perseus/src/util.ts#L273 does correctness checking).